web3_proxy/src/config.rs

```diff
@@ -94,6 +94,14 @@ pub struct AppConfig {
     /// None = allow all requests
     pub default_user_max_requests_per_period: Option<u64>,
 
+    /// Default ERC address for out deposit contract
+    pub deposit_contract: String,
+
+    /// Default ERC address for out deposit contract
+    pub deposit_topic: String,
```

Comment:
As above, this should be an Option. And I think we can use ether's H256 type on this one